### ISSUE TITLE
Clean up fields to display in search results

### ISSUE THREAD

**Issue by ClaireValdivia** [usdigitalresponse/usdr-gost#2107]
### Why is this issue important?

after new search was implemented, the fields in the results table (as well as the excel export) were not updated to reflect new fields or remove fields that are not relevant to users. 

### Current State

The following fields show on My Grants and Browse Grants results table:
- Title
- Viewed by
- Interested Agencies 
- Status - (shows "Opportunity Status")
- Opportunity Category
- Cost Sharing
- Award Floor
- Award Ceiling 
- Open Date 
- Close Date 

The export file includes all these fields, as well as:
- Opportunity Number
- Agency Code
- Grant ID
- URL (this is the grants.gov url)

### Expected State

Drafted:
Assumptions from 10/17 discussion:
- export should match results table in app
- results table should include all searchable fields

The following fields are on the search but don't display on results:
- Opportunity # 
- Funding Type
- Eligibility
- Appropriations Bill
- Agency Code

Remove the following fields from display:
- TBD 

 

### Implementation Plan

- Assess if any of the fields displaying on the results page can be removed (i.e. Viewed by, Interested Agencies, Award Floor, Open Date)
- Confirm fields on search that are not included as a field on results
- Match export to reflect what is in results display table

Once requirements are finalized, need to add any notes on technical approach

**Comment by replicantSocks** [comment]
For eligibility, each grant has multiple eligibility codes, sometimes even as many as 15. Is the proposal to map these to their corresponding phrases e..g "Public housing authorities/Indian housing authorities" and have a column with each of those phrases in a delimited list or is there another way to display Eligibility more succinctly?

**Comment by ClaireValdivia** [comment]
@replicantSocks yes, having the mapped phrases show in a delimited list is the proposal. The phrases should match what is displayed on the dropdown in Search for the Eligibility field

**Comment by ClaireValdivia** [comment]
tested in staging and export looks great!
